src/Card/_variables.scss

```diff
@@ -53,4 +53,4 @@ $loading-skeleton-spacer:            .313rem !default;
 
 $card-focus-border-offset:           5px !default;
 $card-focus-border-width:            2px !default;
-$card-focus-border-radius:           5px !default;
+$card-focus-border-radius:           .625rem !default;
```

Comment:
Should we instead inherit this value from `$card-border-radius` variable somehow? So that we don't have this issue in the future if someone changes base border-radius of the `Card`.

Reply:
They don't equal to each other, `$border-radius: .375rem;`, here we have `.625rem`. I played around to come up with some formula. For example, right now card `$border-radius: .375rem;`, `$card-focus-border-radius:  .625rem` and distance between card and focus is `5px`. In the result to calculate corresponding radius for `focus-border` we need `$border-radius` + `distance between focus-border and card-border`. Check if this will work:
```
$card-focus-border-radius: calc($card-focus-border-offset + $card-border-radius); !default

Reply:
Ok, thanks